### ISSUE TITLE
Move away from nwb

### ISSUE THREAD

**Issue by heatherbooker** [kingofthestack/react-chat-window#93]
It's too magic. What is it even doing??

(nwb being the build tool.)

**Comment by dharness** [comment]
Yeah I don't like it either.. it makes it difficult to advise people on how to fix build issues if we don't control the build process.

**Comment by dharness** [comment]
Let's queue this up for the next release
